multitask/multitask_tutorial.py

```diff
@@ -29,82 +52,59 @@
 # Our data points are 2D points in a square centered on the origin.
 # Our tasks will be classifying whether these points are:
 #
-# 1. Inside a **unit circle** centered on the origin
-# 2. Inside a **unit square** centered on the origin
+# 1. Inside a **unit circle** centered on the origin (label 0 = `False`, label 1 = `True`)
+# 2. Inside a **unit square** centered on the origin (label 0 = `False`, label 1 = `True`)
 #
 # We'll visualize these decision boundaries in a few cells.
 #
-# _Note: In this toy example, we don't expect these specific tasks to help each other learn, but this is often a benefit of joint training in MTL settings._
+# _Note: We don't expect these specific toy tasks to necessarily improve one another, but this is often a benefit of joint training in MTL settings when a model is trained on similar tasks._
+# %%
+import os
 
+# Make sure we're running from the spam/ directory
+if os.path.basename(os.getcwd()) == "snorkel-tutorials":
+    os.chdir("multitask")
 # %%
-import numpy as np
+from utils import make_circle_dataset, make_square_dataset
 
-N = 500  # Data points per dataset
+N = 1000  # Data points per dataset
 R = 1  # Unit distance
 
-# Dataset 0 ("circle")
-circle_data = np.random.uniform(0, 1, size=(N, 2)) * 2 - 1
-circle_labels = circle_data[:, 0] ** 2 + circle_data[:, 1] ** 2 < R
+circle_train, circle_valid, circle_test = make_circle_dataset(N, R)
+(circle_X_train, circle_Y_train) = circle_train
+(circle_X_valid, circle_Y_valid) = circle_valid
+(circle_X_test, circle_Y_test) = circle_test
 
-# Dataset 1 ("square")
-square_data = np.random.uniform(0, 1, size=(N, 2)) * 2 - 1
-square_labels = (abs(square_data[:, 0]) < 0.5) * (abs(square_data[:, 1]) < 0.5)
+square_train, square_valid, square_test = make_square_dataset(N, R)
+(square_X_train, square_Y_train) = square_train
```

Comment:
nit: we've primarily been doing object type first (e.g. df_train vs. train_df). So this would be X_train_square